docs/faq.md

```diff
@@ -67,9 +67,17 @@ https://github.com/shorebirdtech/shorebird/issues/739
 Yes. Shorebird is intended to be used from CI systems. We've published a
 guide for [Github Actions](ci/github.md), other CI systems should be similar.
 
-Please don't hestiate to reach out over Github issues or Discord if you
+Please don't hestiate to reach out over GitHub issues or Discord if you
 encounter any issues.
 
+### What types of changes does Shorebird support?
+
+Shorebird can change any Dart code in your application. This includes app code,
```

Comment:
Would it be useful to also clarify that with Dart it also means `pubspec.yaml` and `pubspec.lock` (to make clear that you could also add a Dart only package to your app and Shorebird still updates it)?

Reply:
Updated, thanks!

Reply:
Cool!